system-test/system-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# Stopping the script if any command fails
+set -e
+source bin/env.sh
+
+function cleanup {
+  echo "Removing pulse-system-test from tmp directory"
+  rm  -r /tmp/pulse-system-test
+  # killing log-collector, collection-roller, Alert-engine, Example application.
+  echo "killing service by parent ID"
+  PID=$(ps -o pgid= $$ | pgrep -u $SOLR_USR java)
+  kill -9 $PID
+  echo "kill service end"
+  echo "Sevices are terminated/killed"
```

Comment:
Remove one of these messages, they are redundant.

Reply:
fixed